Plash/AppDelegate.swift

```diff
@@ -247,10 +247,12 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 	func loadURL(_ url: URL?) {
 		webViewError = nil
 
-		guard let url = url else {
+		guard var url = url else {
 			return
 		}
 
+		url = replacePlaceholders(of: url) ?? url
```

Comment:
Should we really continue if the replacement failed? How about we make this method throwing and do the do/catch here instead and then `error.presentModal()`.

Reply:
Done. Since the method can still silently fail, I opted to use `throws -> URL?`.

Reply:
👍🏻